website/app/app.js

```diff
@@ -8,6 +8,7 @@ import Resolver from 'ember-resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from 'website/config/environment';
 import { importSync, isDevelopingApp, macroCondition } from '@embroider/macros';
+import 'ember-a11y-refocus/styles.css';
```

Comment:
this will not work in classic build, you need to move it into scss file, this method only work under Vite build